minecraftServers/Servers/ServerList.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace minecraftServers.Servers;
+
+public class ServerList
```

Comment:
The fields are great! 👍 But I have some question about naming. The `List` prefix is extra, because it's not a list, it's just a class that represent one server